src/java/fr/paris/lutece/plugins/workflow/modules/formspdf/business/FormsPDFTaskTemplateDAO.java

```diff
@@ -0,0 +1,113 @@
+package fr.paris.lutece.plugins.workflow.modules.formspdf.business;
+
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import fr.paris.lutece.plugins.workflow.modules.formspdf.service.FormsPDFPlugin;
+import fr.paris.lutece.util.sql.DAOUtil;
+
+public class FormsPDFTaskTemplateDAO implements IFormsPDFTaskTemplateDAO {
```

Comment:
if all these templates are in public access, FileService could have been used instead of a new DAO ?